pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/LoadManagerShared.java

```diff
@@ -52,7 +52,7 @@
     private static final Set<String> primariesCache = new HashSet<>();
 
     // Cache for shard brokers according to policies.
-    private static final Set<String> sharedCache = new HashSet<>();
+    private static final Set<String> secondary = new HashSet<>();
```

Comment:
The name should be `secondariesCache` like `primariesCache` and the description should be changed, shouldn't it ?
(at least, `shard` may be typo.)

Reply:
yes, make sense. I will fix it.